server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -102,6 +106,16 @@ static class AdminOpts extends ServerUtilOpts {
     boolean force = false;
   }
 
+  @Parameters(
+      commandDescription = "signal the server process to shutdown normally, finishing anything it might be working on, but not starting any new tasks")
+  static class GracefulShutdownCommand {
+    @Parameter(names = {"-h", "--host"}, description = "<host>")
+    String hostname = null;
+
+    @Parameter(names = {"-p", "--port"}, description = "<port>")
+    int port = 0;
```

Comment:
It seems unnecessary to use JCommander to split these into separate options. Why not just do:

```bash
bin/accumulo admin signalShutdown <host:port>
```

No need for a separate `-h` and `-p` option. They go together anyway. It'd be weird to split them up only to pass them as separate params, so that the code combines them again to act upon it.